packages/liveblocks-client/src/__tests__/room.test.ts

```diff
@@ -936,11 +980,19 @@ describe("room", () => {
 
       const unsubscribe = machine.subscribe("others", (o) => (others = o));
 
+      machine.onMessage(
+        serverMessage({
+          type: ServerMsgCode.ROOM_STATE,
+          users: { 1: {} },
+        })
+      );
```

Comment:
These tests are updated to reflect the need for each client to receive both `ROOM_STATE`… 1/2